docs/Getting-Started.md

```diff
@@ -0,0 +1,270 @@
+      registerOwnerProps: function () {
+        // the todoActions.METHOD will take the input to that method and pass it to the dispatcher, where it will broadcast the METHOD to any store listening. If that action is registered with the store, the payload from that action will then be passed to the corresponding actions method.
+        return {
+          add: todoActions.add,
+          remove: todoActions.remove
+        };
+      },
+
+      render: function () {
+        // Create an array of todo components
+        var Todos = this.state.todos.map(function(todo) {
+          return (
+            <Todo todo={todo} />
+          );
+        });
+
+        // Render all of the Todo components and the TodoCreateForm
+        return (
+          <div>
+            <TodoCreateForm />
+            {Todos}
+          </div>
+        );
+      }
+    });
+
+    module.exports = TodoViewOwner;
+```
+
+## Todo.jsx
+`Tux` also provides the ability to build high performance components out of the box. In our Todo application within the individual Todo component, a Todo's text is a `mutableTraits` property, that means that we can check to see if this property has changed when the component will re-render. If it hasn't, we can skip the re-render since we know the component hasn't updated.
+
+A high performance component:
+
+```javascript
+    var React = require('tux/React');
+
+    var Todo = React.createMutableClass({
+      // Here we are specifying that on the props object there is key that we want to watch and see if it changes and that we only want to re-render the component when that specified prop value changes
+      mutableTraits: {
+        props: 'text'
+      },
+
+      propTypes: {
+        todo: React.PropTypes.object.isRequired
+      },
+
+      // nearestOwnerPropTypes allows us to perform prop validation on our nearestOwnerProps
+      nearestOwnerPropTypes: {
+        remove: React.PropTypes.func.isRequired
+      },
+
+      handleRemove: function (e) {
+        e.preventDefault();
+        this.nearestOwnerProps.remove(this.props.todo);
+      },
+
+      render: function () {
+        return (
+          <div>
+            <p>{ this.props.todo.text }</p>
+            <button onClick={this.handleRemove}>Delete</button>
+          </div>
+        );
+      }
+    });
+
+    module.exports = Todo;
+
+```
+
+## TodoCreateForm.jsx
+Another of the opinionated `Tux` components are the Ownee class. They receive their dynamic props through standard `this.props` sharing, and unless it is needed for input form validation. They don't need to manage their own state standardly as they receive their static methods and props from their closest Owner component. The `TodoCreateForm` utilizes this by accessing the `add` method from its nearest owner component, which is the `TodoViewOwner`.
```

Comment:
Remove ...and unless it is needed for input form validation.

Remove They don't need to manage their own state standardly as